cwltool/singularity.py

```diff
@@ -467,11 +479,17 @@ def create_runtime(
         runtime = [
             "singularity",
             "--quiet",
-            "exec",
             "--contain",
             "--ipc",
             "--cleanenv",
         ]
```

Comment:
This three flags are valid for `singularity exec` or `singularity run`, but not plain `singularity`, so they have to be added after the `exec` or the `run`